metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -1147,11 +1147,17 @@ def fetch_projects():
                 )
             # Set parent project name
             for _project_id, project in all_project_map.items():
-                if (
-                    project.parent_id is not None
-                    and project.parent_id in all_project_map
-                ):
+                if project.parent_id is None:
+                    continue
+
+                if project.parent_id in all_project_map:
                     project.parent_name = all_project_map[project.parent_id].name
+                else:
+                    self.report.warning(
+                        "Incomplete project hierarchy",
+                        f"Parent project {project.parent_id} missed. We need Site Administrator Explorer permissions. Therefore, removing references from child projects.",
+                    )
```

Comment:
```suggestion
                    self.report.warning(
                        "Project details missing. Child projects will be ingested without reference to parent project. We need Site Administrator Explorer permissions to extract complete project hierarchy",
                        project.parent_id,
                    )
```
In favour of less verbose UI warnings.

Will this change remove the need of `if cur_proj.parent_id not in all_project_map:` check from `set_project_path` ?